apps/nextjs-website/package.json

```
@@ -28,6 +30,15 @@
     "slick-carousel": "^1.8.1"
   },
   "devDependencies": {
+    "@chromatic-com/storybook": "^1.3.5",
+    "@storybook/addon-essentials": "^8.1.0",
+    "@storybook/addon-interactions": "^8.1.0",
+    "@storybook/addon-links": "^8.1.0",
+    "@storybook/addon-onboarding": "^8.1.0",
+    "@storybook/blocks": "^8.1.0",
+    "@storybook/nextjs": "^8.1.0",
+    "@storybook/react": "^8.1.0",
+    "@storybook/test": "^8.1.0",
```

```suggestion
    "@storybook/nextjs": "^8.1.0",
    "@storybook/react": "^8.1.0",
```
I don't see the two removed used anywhere